src/components/BalanceButtons.vue

```diff
@@ -37,7 +36,7 @@ export default {
        * Emitted when send button is clicked
        * @event send
        */
-      if (this.currentStatus === NETWORK_STATUS.SYNCED) {
+      if (this.currentStatus === this.$t('synced')) {
```

Comment:
We should only use translations in the view. If we don't have a translation for currentStatus the sentence could fail in the future.